.azure_pipelines/olive-ci.yaml

```diff
@@ -29,9 +29,10 @@ variables:
 jobs:
 - template: job_templates/olive-build-template.yaml
   parameters:
-    name: Linux_CI
+    name: Linux_CI_latest_ORT
```

Comment:
For the latest test, can we schedule it weekly? or trigger it manually? Otherwise, in the period of no ort releasing, this CI jobs have duplicated tests.

Reply:
This is not for ORT nightly. it is for latest release ORT.

Reply:
Emm, then where is the nightly build? (I just thought you are adding the nightly build tests from the PR title.